docs/source/user-guide/index.rst

```diff
@@ -1,5 +1,5 @@
 ==========
-User guide
+User Guide
```

Comment:
Any reason for this capitalization here? The other page titles only capitalize the first word.

Reply:
```suggestion
User guide
```

Reply:
No reason. Let's go with the suggestion.